pkg/model/components/etcdmanager/options.go

```diff
@@ -57,12 +57,7 @@ func (b *EtcdManagerOptionsBuilder) BuildOptions(o interface{}) error {
 		}
 
 		if etcdCluster.Version == "" {
-			if b.IsKubernetesGTE("1.11") {
-				etcdCluster.Version = "3.2.18"
-			} else {
-				// Preserve existing default etcd version
-				etcdCluster.Version = "2.2.1"
```

Comment:
Remove `2.2.1` also from `supportedEtcdVersions`.

Reply:
Cannot. Used as default for legacy provider with k8s 1.11 and 1.12

Reply:
Ah.. Thanks